### PR TITLE
[Docs][Connector-V2][Doris] Reconstruct the Doris connector document

### PR DESCRIPTION
**Pull request by Carl-Zhou-CN**
subtask of #4572 

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/Doris.md [hunk 1]
```diff
@@ -2,11 +2,24 @@
 
 > Doris sink connector
 
+## Support Those Engines
+
+> Spark<br/>
+> Flink<br/>
+> Seatunnel Zeta<br/>
+
+## Key features
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Doris.md]: Key Features

File: docs/en/connector-v2/sink/Doris.md [hunk 2]
```diff
@@ -17,67 +30,96 @@ Version Supported
...
+      read_json_by_line = "true"
+    }
+  }
+}
 
-## Example
+```
 
-Use JSON format to import data
+### Use JSON format to import data
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Doris.md:212]: Why are there two?

File: docs/en/connector-v2/sink/Doris.md (hunk 1 above)
Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md]: done

File: docs/en/connector-v2/sink/Doris.md (hunk 2 above)
Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md:212]: "CSV and JSON, respectively.

File: docs/en/connector-v2/sink/Doris.md [hunk 3]
```diff
@@ -17,67 +30,96 @@ Version Supported
...
 
-The supported formats include CSV and JSON. Default value: CSV
+sink {
+  Doris {
+    fenodes = "doris_cdc_e2e:8030"
+    username = root
+    password = ""
+    table.identifier = "test.e2e_table_sink"
+    sink.label-prefix = "test-cdc"
+    sink.enable-2pc = "false"
```
Comment (zy-kkk) [inline docs/en/connector-v2/sink/Doris.md]: 2pc is a nice feature, we should enable it by default in our examples, and we need to add the checkpoint interval to env

Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md]: done

**Comment by Carl-Zhou-CN** [comment]
@zy-kkk @zhilinli123 @TyrantLucifer CC,Thanks

File: docs/en/connector-v2/sink/Doris.md [hunk 4]
```diff
@@ -17,67 +30,97 @@ Version Supported
...
+  job.mode = "BATCH"
+  checkpoint.interval = 10000
+}
 
-#### Supported import data formats
+source {
+  FakeSource {
+    schema = {
+      fields {
+        pk_id = bigint
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Doris.md:154]: Add all the data types, it's more friendly

**Comment by EricJoy2048** [comment]
You lost some thing like this:

image

**Comment by Carl-Zhou-CN** [comment]
@EricJoy2048 added data type mapping

File: docs/en/connector-v2/sink/Doris.md (hunk 4 above)
Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md:154]: I have added as many types as possible that are supported by all versions

File: docs/en/connector-v2/sink/Doris.md [hunk 5]
```diff
@@ -17,67 +30,130 @@ Version Supported
...
-### sink.label-prefix [string]
-
-The label prefix used by stream load imports. In the 2pc scenario, global uniqueness is required to ensure the EOS semantics of SeaTunnel.
-
-### sink.enable-2pc [bool]
-
-Whether to enable two-phase commit (2pc), the default is true, to ensure Exactly-Once semantics. For two-phase commit, please refer to [here](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD).
-
-### sink.enable-delete [bool]
+#### Supported import data formats
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Doris.md:76]: `Supported Import Data Formats`?

File: docs/en/connector-v2/sink/Doris.md [hunk 6]
```diff
@@ -17,67 +30,130 @@ Version Supported
...
-https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Manipulation-Statements/Load/STREAM-LOAD
+```hocon
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+  checkpoint.interval = 10000
+}
 
-#### Supported import data formats
+source {
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/Doris.md:150]:
```
FakeSource {
  row.num = 10
  map.size = 10
  array.size = 10
  bytes.length = 10
  string.length = 10
  schema = {
    fields {
      c_map = "map<string, array<int>>"
      c_array = "array<int>"
      c_string = string
      c_boolean = boolean
      c_tinyint = tinyint
      c_smallint = smallint
      c_int = int
      c_bigint = bigint
      c_float = float
      c_double = double
      c_decimal = "decimal(30, 8)"
      c_null = "null"
      c_bytes = bytes
      c_date = date
      c_timestamp = timestamp
      c_row = {
        c_map = "map<string, map<string, string>>"
        c_array = "array<int>"
        c_string = string
        c_boolean = boolean
        c_tinyint = tinyint
        c_smallint = smallint
        c_int = int
        c_bigint = bigint
        c_float = float
        c_double = double
        c_decimal = "decimal(30, 8)"
        c_null = "null"
        c_bytes = bytes
        c_date = date
        c_timestamp = timestamp
      }
    }
  }
  result_table_name = "fake"
}

```
This data source type is more comprehensive

Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md:150]: Not every version of doris supports complex types

File: docs/en/connector-v2/sink/Doris.md (hunk 5 above)
Comment (Carl-Zhou-CN) [reply docs/en/connector-v2/sink/Doris.md:76]: This is the original category and can be deleted if not needed

File: docs/en/connector-v2/sink/Doris.md (hunk 6 above)
Comment (zhilinli123) [reply docs/en/connector-v2/sink/Doris.md:150]: PTAL: @TaoZex @EricJoy2048

**Review by EricJoy2048 — APPROVED** [review]

**Review by TaoZex — APPROVED** [review]
+1

**Review by hailin0 — APPROVED** [review]

**Pull request merged**